trunner/target/emulated.py

```diff
@@ -22,7 +27,9 @@ def __init__(self, script: str):
         super().__init__()
         self.script = script
         # TODO Make sure that script path exists
-        self.dut = QemuDut(f"{self.project_dir}/scripts/{self.script}", encoding="utf-8")
+        self.dut = QemuDut(
+            f"{self.project_dir}/scripts/{self.script}", encoding="utf-8"
+        )
```

Comment:
Nit: For future: changes about  formatting of untouched parts of  file should not be part of the same pr